sqlglot/diff.py

```diff
@@ -279,7 +286,9 @@ def _expression_only_args(expression):
     return [a for a in args if isinstance(a, exp.Expression)]
 
 
-def _lcs(seq_a, seq_b, equal):
+def _lcs(
```

Comment:
This function was a bit annoying to annotate completely and remove the ignore cheat..